File: 1684_leetcode.go
func countConsistentStrings(allowed string, words []string) int {
	g := make([]bool, 27)
	for _, c := range allowed {
		g[int(c-'a')] = true
	}

	ans := 0
	for _, s := range words {
		good := true
		for _, c := range s {
			if !g[int(c-'a')] {
				good = false
				break
			}
		}
		if good {
			ans++
		}
	}

	return ans
}

File: 1684_leetcode.py
class Solution:
    def countConsistentStrings(self, allowed: str, words: List[str]) -> int:
        all_set = set(allowed)
        counter = 0
        for i in words:
            flag = True
            for j in i:
                if j not in all_set:
                    flag = False
                    break
            if flag:
                counter += 1
        return counter


class Solution:
    def countConsistentStrings(self, allowed: str, words: List[str]) -> int:
        all_set = set(allowed)
        counter = 0
        for i in words:
            counter += 0 if len(set(i)-all_set) else 1
        return counter